conf/iCubFindDependencies.cmake

```diff
@@ -65,8 +65,8 @@ checkandset_dependency(OpenCV)
 checkandset_dependency(Qt5)
 
 if(icub_firmware_shared_FOUND AND ICUB_USE_icub_firmware_shared)
-  if(icub_firmware_shared_VERSION VERSION_LESS 1.35.0)
-    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected: at least 1.35.0 is required")
+  if(icub_firmware_shared_VERSION VERSION_LESS 1.35.1)
+    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected: at least 1.35.1 is required")
```

Comment:
This is strange, as the version is already ok in `icub-main@devel`.

https://github.com/robotology/icub-main/blob/17decbc8f911ae3d1664eca8422bf261f3b46292/conf/iCubFindDependencies.cmake#L67-L71

It has been recently updated via #886.

I recommend that you revert the file, just in case.

Reply:
I've seen that you increased the patch number 👍🏻 
Solved.